test/correctness/dynamic_allocation_in_gpu_kernel.cpp

```diff
@@ -4,41 +4,51 @@ using namespace Halide;
 
 int main(int argc, char **argv) {
     Target t(get_jit_target_from_environment());
-    if (!t.has_gpu_feature()) {
+    if (!t.has_gpu_feature() || t.has_feature(Target::OpenGLCompute)) {
```

Comment:
Is this `t.has_feature(Target::OpenGLCompute` intentional?

Reply:
Yes, OpenGLCompute doesn't otherwise count as has_gpu_feature because it has such a terrible type system.

Reply:
Yes but I believe this conditional is wrong-- I think you _do_ want to run the test for OpenGLCompute.

Reply:
oops

Reply:
Fixed.